packages/modular-scripts/react-scripts/config/webpack.config.js

```diff
@@ -360,35 +356,15 @@ module.exports = function (webpackEnv) {
           // match the requirements. When no loader matches it will fall
           // back to the "file" loader at the end of the loader list.
           oneOf: [
-            // Required since esbuild loader will not pass svg the same as babel-loader
-            // was handling them
-            {
-              test: /\.svg$/,
-              use: [
-                require.resolve('@svgr/webpack'),
-                require.resolve('url-loader'),
-              ],
-            },
             // TODO: Merge this config once `image/avif` is in the mime-db
             // https://github.com/jshttp/mime-db
```

Comment:
I'm quite certain that image/avif is in the mime-db in https://github.com/jshttp/mime-db, but didn't want to go out of scope and remove this line in this PR.